src/wiki/plugins/editsection/apps.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import unicode_literals
```

Comment:
You can remove this due to Python 3 only.

Reply:
Done for all of them.